packer/templates/monitoring/prometheus/prometheus.yml

```diff
@@ -0,0 +1,28 @@
+global:
+  scrape_interval:     5s # By default, scrape targets every 15 seconds.
+  # scrape_timeout is set to the global default (10s).
+  external_labels:
+      monitor: 'perf-test'
+
+scrape_configs:
+  - job_name: 'prometheus'
+    scrape_interval: 5s
+    static_configs:
+         - targets: ['localhost:9090']
+
+  - job_name: 'cadvisor'
+    scrape_interval: 5s
+    static_configs:
+         - targets: ['cadvisor:8080']
+
+  - job_name: 'fb-baseline'
+    scrape_interval: 5s
+    metrics_path: /api/v1/metrics/prometheus
+    static_configs:
+         - targets: ['fb-baseline:2020']
+
+  - job_name: 'fb-delta'
+    scrape_interval: 5s
+    metrics_path: /api/v1/metrics/prometheus
+    static_configs:
+         - targets: ['fb-delta:2020']
```

Comment:
Configuring this with prometheus as an option is maybe something we want to do later...

If we provide a custom compose stack then we possibly can use the sd_dns_config approach to monitor for new compose services and just scrape them? TODO: investigate.